api/tests/unit/integrations/github/test_unit_github_views.py

```diff
@@ -1044,3 +1055,78 @@ def test_send_the_invalid_type_page_or_page_size_param_returns_400(
     assert response.status_code == status.HTTP_400_BAD_REQUEST
     response_json = response.json()
     assert response_json == error_response
+
+
+# Unit test for enable_linked_feature view in feature_external_resources views
+def test_enable_linked_feature(
+    admin_client_new: APIClient,
+    organisation: Organisation,
+    project: Project,
+    environment: Environment,
+    github_configuration: GithubConfiguration,
+    github_repository: GithubRepository,
+    feature_external_resource: FeatureExternalResource,
+) -> None:
+    # Given
```

Comment:
```suggestion
def test_enable_linked_feature(
    admin_client_new: APIClient,
    organisation: Organisation,
    project: Project,
    environment: Environment,
    github_configuration: GithubConfiguration,
    github_repository: GithubRepository,
    feature_external_resource: FeatureExternalResource,
) -> None:
    """Unit test for enable_linked_feature view in feature_external_resources views"""
    # Given
```

Reply:
I have to remove this part since it will be out of scope for now. It will be included in a new PR in the future.